README.md

```diff
@@ -1 +1,187 @@
-# capstone
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
```

Comment:
- [ OPTIONAL] I would advise you to remove the logo because the image is broken and it doesn't make your README.md file look attractive. This is the way it looks:

![image](https://user-images.githubusercontent.com/77106962/205458603-92fe65b5-f83a-4fcf-970a-8232f5046c27.png)